### ISSUE TITLE
Feature needed as Application Parts in asp.net core, allows multiple GRPC services to be loaded in one process.

### ISSUE THREAD

**Issue by mobwiz** [grpc/grpc-dotnet#1325]
### Is your feature request related to a problem? Please describe.
Our team use the GRPC services to build micro services system. But when we need to deploy this system to a single server for test, It's difficult to configure the proxy such as envoy or nginx. 

We found that the Asp.net core has the "Application Part" which allows developers to build WebApi controllers separately and use  the application part to load them together within one process.

Which like : https://docs.microsoft.com/en-us/aspnet/core/mvc/advanced/app-parts?view=aspnetcore-5.0 

And we can not found similar ability  for GRPC service applications.

If we can use reflection to load an assembly which has implemented some Grpc Services, and aggregate them in one process. Those grpc services can be developed separately, and be loaded together or separately according to the deployment scenario.

### Describe the solution you'd like
I'd like this solution:
1. Use the reflection to load an assembly which contains some grpc services.
2. Use some technique such as "Application Part" in asp.net core to load those grpc services and map the route.
3. Use this single grpc service process to serve these services.

**Comment by BrennanConroy** [comment]
Dupe of https://github.com/grpc/grpc-dotnet/issues/1202
TLDR; we aren't going to be adding scanning to the library.

**Comment by JamesNK** [comment]
Closing as answered.
